docs/docs/platform/multiple-environments.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Multiple Environments'
+sidebar_position: 3
+---
+
+Here's a guide on how to use Nhost with multiple environments and how to set up a workflow around them.
+
+For this example, we'll set up one **production environment** and one **staging environment**.
+
+## Git
+
+Create a new Git repo and use the [CLI](/cli) and the `nhost init` command to initialize a new project.
+
+Also, create a new branch called `staging`.
+
+You should now have a Git repo with a `main` branch and a `staging` branch.
+
+## Nhost Cloud
+
+To have two environments, we need to create two projects on Nhost Cloud.
```

Comment:
An image explaining this structure should be added to this section.

Reply:
That could help.

Could you sketch something on your iPad or something and post here? Curious to see what you're thinking.

Reply:
I was thinking of a screenshot showing that you have two projects. Just to support the text with some visual content as well.

Reply:
Will do it in follow-up PR.